states/new_jersey/elections.toml

```diff
@@ -3,7 +3,140 @@ date = 2020-07-07
+sources = [
+    "https://www.state.nj.us/state/elections/primary-election-070720.shtml",
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"
+]
+
+[20200707.absentee.military_overseas.application]
+online_by = "2020-07-03"
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.absentee]
+postmarked_by = 2020-07-07
+received_by = 2020-07-14T20:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/election-results/2020/2020-drop-box-locations.pdf",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-02-primary-election-052220.pdf"]
+
+[20200707.poll]
+in_person_start = 2020-07-07T06:00:00
+in_person_by = 2020-07-07T20:00:00
+sources = ["https://elections.delaware.gov/calendar/2020ElectionCalendar.pdf",
+           "https://lis.njleg.state.nj.us/nxt/gateway.dll/statutes%2F1%2F19539%2F19749"]
+
+[20200929]
+date = 2020-09-29
+name = "September Special School Board"
+original_date = 2020-09-29
+
+[20200929.registration]
+received_by = 2020-09-08
+in_person_by = 2020-09-08
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"]
+
+[20200929.absentee.application]
+received_by = 2020-09-29
+in_person_by = 2020-09-28T15:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"
+]
+
+[20200929.absentee.military_overseas.application]
+online_by = 2020-09-25
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"
+]
+
+[20200929.absentee]
+postmarked_by = 2020-09-29
+in_person_by = 2020-09-29T20:00:00
+received_by = 2020-10-01T20:00:00
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf"]
+
+[20200929.poll]
+in_person_start = 2020-09-29T16:00:00
+in_person_by = 2020-09-29T20:00:00
+sources = ["https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-08-september-school-board.pdf",
+           "https://lis.njleg.state.nj.us/nxt/gateway.dll/statutes%2F1%2F19539%2F19749"]
+
 [20201103]
 date = 2020-11-03
 name = "General"
 original_date = 2020-11-03
+
+[20201103.registration]
+received_by = 2020-10-13
+in_person_by = 2020-10-13
+sources = ["https://www.state.nj.us/state/elections/voter-registration.shtml",
+           "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-general-election-040820.pdf"]
+
+[20201103.absentee.application]
+received_by = 2020-10-27
+in_person_by = 2020-11-02T15:00:00
+sources = [
+    "https://www.state.nj.us/state/elections/assets/pdf/chrons/2020-chron-general-election-040820.pdf"
+]
+
+[20201103.absentee.military_overseas.application]
+online_by = "2020-07-03"
```

Comment:
online_by = 2020-10-30

From Sources: 
October 30  Deadline for Application to Receive General Election Mail-In Ballots by Electronic Means for Qualified Overseas Civilian and Military Voters(4th day before election)